README.md

```diff
@@ -58,6 +58,7 @@ docker run bcgsc/pori-graphkb-loader --help
 - [COSMIC](./src/cosmic)
 - [DGIdb](./src/dgidb)
 - [DoCM](./src/docm)
+- [MOAlmonac](./src/moa)
```

Comment:
also here

Reply:
I think you might be looking at an old view that doesn't include the fix commit? On mine it is showing up as changed already